src/js/utils/DateUtil.ts

```diff
@@ -135,6 +135,15 @@ const DateUtil = {
     formatKey: moment.DurationInputArg2 = "seconds"
   ): string {
     return moment.duration(time, formatKey).humanize();
+  },
+
+  isValidDate(dateString: string) {
```

Comment:
I think this is not really right regarding typescript.
You want to express "I get something thats a string or null", otherwise you would not check for null. It should be `datestring?: string`, imho

Reply:
I don't think `dateString?` is right, I think it should be `dateString: string | null`.

Reply:
Yeah, but could it also be undefined? Do we need `dateString?: string | null`

Reply:
Please keep in mind that `datestring?: string` is different from `datestring: string | undefined` in that it allows the call without an argument (`isValidDate()`) which does not make a lot of sense here.

Reply:
Yeah right, thanks for adding @pierrebeitz 👍

Reply:
I agree with `Maybe<string>`, but the Maybe PR is not yet merged. So, for now, are we okay with `string | null`?  @pierrebeitz @DanielMSchmidt

Reply:
👍